src/field_theory/separable.lean

```diff
@@ -447,6 +448,16 @@ begin
   exact_mod_cast (enat.add_one_le_of_lt hq)
 end
 
+lemma separable.squarefree {p : polynomial F} (hp : p ≠ 0)  (hsep : separable p) :
+  squarefree p :=
+begin
+  apply (multiplicity.squarefree_iff_multiplicity_le_one p).2,
+  intro f,
+  by_cases hunit : is_unit f,
+  { exact or.inr hunit },
+  exact or.inl (multiplicity_le_one_of_seperable hunit hsep)
+end
+
 lemma root_multiplicity_le_one_of_seperable {p : polynomial F} (hp : p ≠ 0)
```

Comment:
```suggestion
lemma root_multiplicity_le_one_of_separable {p : polynomial F} (hp : p ≠ 0)
```

Reply:
And similar typo change in the lemmas and body proof above.

Reply:
I've already submitted a separate PR for that, FWIW